lib/v2/screens/profile_screens/security/components/guardian_security_card.dart

```diff
@@ -0,0 +1,125 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+import 'package:seeds/v2/components/notification_badge.dart';
+import 'package:seeds/v2/constants/app_colors.dart';
+import 'package:seeds/v2/components/divider_jungle.dart';
+import 'package:seeds/v2/domain-shared/ui_constants.dart';
+import 'package:seeds/v2/design/app_theme.dart';
+import 'package:seeds/i18n/security.i18n.dart';
+import 'package:seeds/v2/screens/profile_screens/security/interactor/viewmodels/security_state.dart';
+
+class GuardianSecurityCard extends StatelessWidget {
+  final GuardiansStatus? guardiansStatus;
+
+  final GestureTapCallback? onTap;
+
+  final bool hasNotification;
+
+  const GuardianSecurityCard({Key? key, this.guardiansStatus, this.onTap, this.hasNotification = false})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    Widget guardianStatus = Container(height: 16, width: 16, child: const Center(child: CircularProgressIndicator()));
+
+    if (guardiansStatus == GuardiansStatus.active) {
```

Comment:
do a switch , instead of multiple if else

Reply:
done